plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/DragAndDropFromTableAndTreeToDiagramTest.java

```diff
@@ -184,6 +184,8 @@ protected void doExecute() {
     protected void tearDown() throws Exception {
         // Reopen outline
         designerViews.openOutlineView();
+        editor.close();
```

Comment:
Already `null` here:
```
java.lang.NullPointerException: Cannot invoke "org.eclipse.sirius.tests.swtbot.support.api.editor.SWTBotSiriusDiagramEditor.close()" because "this.editor" is null
	at org.eclipse.sirius.tests.swtbot.DragAndDropFromTableAndTreeToDiagramTest.tearDown(DragAndDropFromTableAndTreeToDiagramTest.java:187)
```

Reply:
Now fixed.